src/components/logical-expression-builder/interfaces/DisplayValueFunction.ts

```diff
@@ -0,0 +1 @@
+export type DisplayValueFunction = (value: any, fieldType?: string, fieldData?: any) => string;
```

Comment:
Users can provide this function to override the format function in the DisplayValuePipe to customize how the entered value is displayed in static mode.

Reply:
Because in our last meeting with UX Aspects and Timo, we talked about how to customize what is displayed. And if I remember correctly we said for the first iteration it is okay to just display as a text. Also, the implementation was pretty straight-forward and not very much effort.